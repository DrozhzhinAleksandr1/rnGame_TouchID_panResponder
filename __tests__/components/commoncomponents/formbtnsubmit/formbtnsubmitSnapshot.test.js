import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FormBtnSubmit from '../../../../app/components/commoncomponents/formbtnsubmit/formbtnsubmit';

it('create snapshot Input', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FormBtnSubmit />);
    expect(toolbarSnapshot).toMatchSnapshot();
});