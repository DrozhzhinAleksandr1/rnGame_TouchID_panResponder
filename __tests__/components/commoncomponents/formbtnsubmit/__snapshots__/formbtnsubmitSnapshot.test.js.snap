// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Input 1`] = `
<TouchableWithoutFeedback
  disabled={true}
>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 56,
          "justifyContent": "center",
          "marginBottom": 15,
          "paddingLeft": 15,
          "paddingRight": 15,
          "width": 296,
        },
        Object {
          "backgroundColor": "#1D2E3E",
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontFamily": "Roboto-Light",
            "fontSize": 22,
          },
          Object {
            "color": "#5A6772",
          },
        ]
      }
    />
    <ShowSpiner />
  </Component>
</TouchableWithoutFeedback>
`;
